### DISCUSSION TITLE
[Feature] self-service password reset

### DISCUSSION THREAD

**Discussion by Havouza15** [immich-app/immich#6748] [Feature Request]
### The feature

Hi!
is the self service password reset in the pipeline. We are a group on friends that use my immich servers hosted on a vps to avoid Google Photo. Sometimes someone forget the password and I know I as admin can reset it but I am not always available so the self service would be nice

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The built-in auth is deliberately very limited to keep things simple. Password reset would require implementing sending of emails, etc, so I think we're unlikely to add that. If the built in auth system doesn't work for you, we recommend using OAuth instead.

**Reply by Havouza15** [reply]
Well not everyone is familiar with 2 step verification. But ok I understand

**Reply by bo0tzz** [reply]
You can use OAuth without 2FA.
